content/posts/irods-development-update-august-2024.md

```diff
@@ -0,0 +1,260 @@
+
+On the packaging side of things, plugins and clients provided by the Consortium will now include the version of iRODS used to compile the binaries. For example, the Logical Quotas rule engine plugin for iRODS 4.3.3 / Ubuntu 22.04 will have a package name **irods-rule-engine-plugin-logical-quotas_4.3.3.0-0+4.3.3~jammy_amd64.deb**. This is the first step towards independent versioning for plugins and clients. We'll have more to share as things develop.
+
+The PAM Interactive Authentication plugin has received a good number of improvements since last month. Issues with TLS/SSL and account spoofing _(thanks Leiden University)_ have been resolved. The plugin now uses configuration from the **R_GRID_CONFIGURATION** table just like the PAM Password Authentication plugin. And, we've successfully demonstrated PAM Password and Kerberos flows using this plugin. We're very close to a first release so stay tuned.
+
+Users of the Python iRODS client (PRC) will be happy to know that 2.1.0 has been released. To learn what's changed since 2.0.0, see the release notes at [https://github.com/irods/python-irodsclient/releases/tag/v2.1.0](https://github.com/irods/python-irodsclient/releases/tag/v2.1.0).
+
+Work on the next release of the iRODS HTTP API is going well. The parser responsible for handling multipart/form-data has been rewritten to handle embedded CRLFs. We've also started to lay the foundation for user mapping plugins. The minimum version requirement for compiling the HTTP API is being bumped to iRODS 4.3.2. The reason for this is due to iRODS 4.3.2 providing built-in support for GenQuery2. This ultimately means the GenQuery2 API plugin will not be supported in the next release.
+
+Multipart upload performance in the iRODS S3 API has been improved. We're very happy with how things are progressing, but there's still a few things to work out before this lands. If you find that interesting, check out the pull request at [https://github.com/irods/irods_client_s3_api/pull/115](https://github.com/irods/irods_client_s3_api/pull/115).
+
+For the Zone Management Tool (ZMT), our intern has successfully replaced all use of the iRODS C++ REST API with the iRODS HTTP API. That work will land very soon.
+
+It was brought to our attention that the Globus Connector was not available for iRODS 4.3.2. That has since been corrected. Packages are now availables via the packages repository.
```

Comment:
```suggestion
It was brought to our attention that the Globus Connector was not available for iRODS 4.3.2. That has since been corrected. Packages are now available via the packages repository.
```

Reply:
Done.